util/envDynawo.sh

```diff
@@ -1454,86 +1484,113 @@ deploy_dynawo() {
+    sed -i "s|$DYNAWO_XERCESC_INSTALL_DIR|$DYNAWO_DEPLOY_DIR|" $file
   done
-  for file in $(grep -rl $DYNAWO_LIBIIDM_HOME extraLibs/LIBIIDM/share/cmake/*); do
-    sed -i "s|$DYNAWO_LIBIIDM_HOME|$DYNAWO_DEPLOY_DIR/extraLibs/LIBIIDM|" $file
+  for file in $(grep -rl $DYNAWO_LIBIIDM_HOME share/cmake/*); do
+    sed -i "s|$DYNAWO_LIBIIDM_HOME|$DYNAWO_DEPLOY_DIR|" $file
   done
-  for file in $(grep -rl $DYNAWO_LIBXML_HOME extraLibs/LIBIIDM/share/cmake/*); do
-    sed -i "s|$DYNAWO_LIBXML_HOME|$DYNAWO_DEPLOY_DIR/extraLibs/LIBXML|" $file
+  for file in $(grep -rl $DYNAWO_LIBXML_HOME share/cmake/*); do
+    sed -i "s|$DYNAWO_LIBXML_HOME|$DYNAWO_DEPLOY_DIR|" $file
   done
 
-  mkdir -p 3rdParty/openmodelica/bin/
-  mkdir -p 3rdParty/openmodelica/include/
-  mkdir -p 3rdParty/openmodelica/lib/omc/
-  cp -P $DYNAWO_INSTALL_OPENMODELICA/bin/omc* 3rdParty/openmodelica/bin
-  cp -P -R $DYNAWO_INSTALL_OPENMODELICA/include/omc 3rdParty/openmodelica/include/
-  cp -P -R $DYNAWO_INSTALL_OPENMODELICA/lib/* 3rdParty/openmodelica/lib/
-  cp -P $DYNAWO_INSTALL_OPENMODELICA/lib/omc/*.mo 3rdParty/openmodelica/lib/omc/
-  cp -P -R $DYNAWO_INSTALL_OPENMODELICA/lib/omlibrary 3rdParty/openmodelica/lib/
+  mkdir -p OpenModelica/bin/
+  mkdir -p OpenModelica/include/
+  mkdir -p OpenModelica/lib/omc/
+  cp -P $DYNAWO_INSTALL_OPENMODELICA/bin/omc* OpenModelica/bin
+  cp -P -R $DYNAWO_INSTALL_OPENMODELICA/include/omc OpenModelica/include/
+  cp -P -R $DYNAWO_INSTALL_OPENMODELICA/lib/* OpenModelica/lib/
+  cp -P $DYNAWO_INSTALL_OPENMODELICA/lib/omc/*.mo OpenModelica/lib/omc/
+  cp -P -R $DYNAWO_INSTALL_OPENMODELICA/lib/omlibrary OpenModelica/lib/
+
+  if [ "$DYNAWO_LIBRARY_TYPE" = "SHARED" ]; then
+    LIBRARY_SUFFIX=$DYNAWO_SHARED_LIBRARY_SUFFIX
+  elif [ "$DYNAWO_LIBRARY_TYPE" = "STATIC" ]; then
+    LIBRARY_SUFFIX="a"
+  else
+    error_exit "Wrong type of library build: $DYNAWO_LIBRARY_TYPE, either SHARED or STATIC."
+  fi
 
   # BOOST
   if [ $DYNAWO_BOOST_HOME_DEFAULT != true ]; then
-    cp -P $DYNAWO_BOOST_HOME/lib/libboost_*.$DYNAWO_SHARED_LIBRARY_SUFFIX* extraLibs/BOOST/lib/
-    cp -P -R $DYNAWO_BOOST_HOME/include/boost extraLibs/BOOST/include/
+    boost_system_folder=$DYNAWO_BOOST_HOME/lib
+    boost_system_folder_include=$DYNAWO_BOOST_HOME/include
   else
-    boost_system_folder=$(find_lib_system_path boost)
-    cp -P ${boost_system_folder}libboost_*.$DYNAWO_SHARED_LIBRARY_SUFFIX* extraLibs/BOOST/lib/
-    boost_system_folder_include=$(find_include_system_path Boost_INCLUDE_DIR)
-    cp -P -R $boost_system_folder_include/boost extraLibs/BOOST/include/
-  fi
+    boost_system_folder=$(find_lib_system_path boost) || error_exit "Path for boost could not be found for deploy."
+    boost_system_folder_include=$(find_include_system_path Boost_INCLUDE_DIR) || error_exit "Path for boost include could not be found for deploy."
+  fi
+  cp -P ${boost_system_folder}/libboost_atomic.$LIBRARY_SUFFIX* lib/
```

Comment:
Harder to maintain than the previous implementation (if we had a new boost dependency this will not work anymore)

Reply:
On the other hand we would distribute things we don't need...

Reply:
Is there a way to automatically find this list?

Reply:
I did it